src/library-authoring/components/LibraryComponents.test.tsx

```diff
@@ -163,6 +164,17 @@ describe('<LibraryComponents />', () => {
     expect(screen.queryByRole('button', { name: /add component/i })).not.toBeInTheDocument();
   });
 
+  it('should render a spinner while loading', async () => {
+    mockUseSearchContext.mockReturnValue({
+      ...data,
+      isLoading: true,
+    });
+
+    render(<RootWrapper />);
+    const spinner = await screen.findByRole('status');
+    expect(spinner.textContent).toEqual('Loading...');
+  });
+
```

Comment:
This test is failing when I run it locally, and I've been utterly unable to figure out why. The error I'm seeing is below -- help?

```
  ● <LibraryComponents /> › should render a spinner while loading

    TypeError: Cannot read properties of undefined (reading 'visibility')

    Ignored nodes: comments, script, style
    <body>
      <div>
        <div
          data-testid="redux-provider"
        >
          <div
            data-testid="browser-router"
          >
            <div
              class="pgn__spinner spinner-border text-primary"
              role="status"
            >
              <span
                class="sr-only"
              >
                Loading...
              </span>
            </div>
          </div>
        </div>
      </div>
    </body>

      at isInaccessible (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/role-helpers.js:62:39)
      at node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/queries/role.js:144:63
          at Array.filter (<anonymous>)
      at queryAllByRole (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/queries/role.js:143:6)
      at node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/query-helpers.js:74:17
      at node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/query-helpers.js:52:17
      at node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/query-helpers.js:95:19
      at container.container (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/query-helpers.js:87:14)
      at runWithExpensiveErrorDiagnosticsDisabled (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/config.js:47:12)
      at checkCallback (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/wait-for.js:127:77)
      at checkRealTimersCallback (node_modules/@testing-library/react/node_modules/@testing-library/dom/dist/wait-for.js:121:16)
      at Timeout.task [as _onTimeout] (node_modules/jsdom/lib/jsdom/browser/Window.js:520:19)
```